sandbox/registration-processor-camel-routes-update-mz.xml

```diff
@@ -111,6 +111,7 @@
             message="osi-validator-->biometric-authentication-stage update route ${bodyAs(String)}" />
         <choice>
             <when>
+		<jsonpath suppressExceptions="true">$.[?(@['tags']['ID_OBJECT-presentCity'] != 'Tubo' &amp;&amp; @['isValid'] == true)]</jsonpath>
                 <simple>${bodyAs(String)} contains '"isValid":true'</simple>
```

Comment:
why simple is not removed ?

Reply:
Removed Simple and changed city Tubo to Lacub